ui/app/src/Views/FilterBar/Views.elm

```diff
@@ -122,6 +126,13 @@ view { matchers, matcherText, backspacePressed } =
                         , span
                             [ class "input-group-btn" ]
                             [ button [ class "btn btn-primary", disabled isDisabled, onClickAttr ] [ text "+" ] ]
+                        , a
+                            [ class "btn btn-outline-info border-0"
+                            , href (newSilenceFromMatchers dataMatchers)
+                            ]
+                            [ i [ class "fa fa-bell-slash-o mr-2" ] []
+                            , text "Silence"
+                            ]
```

Comment:
Could you please attach a screenshot to your PR? It would make it easier to review!

Reply:
OK I'll attach the screenshot.